### PR TITLE
#3470 - Content: Student: BCSC

### PR DESCRIPTION
**Pull request by bidyashish**
**Acceptance Criteria**
- [X] **Common to Both** Remove text: "Use your most up to date personal information. We'll use the same information to help you apply for financial aid."  
   Already done
- [x] **Common to Both** Revise text: "Profile Information" to "Profile information:
- [x] **BCSC only** Revise text: Remove "here" from sentence and add "." after "up-to-date."
- [x] **Common to Both** Revise info box header: "How to update your information" to "Information"
- [x] **BCSC only** Revise info box text:  "Your information above is taken from...." to "Your information above is taken from your BC Services Card account. If the information is incorrect, please update your information with [BC Services Card account.](https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/your-card/change-personal-information)

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/studentprofile.json
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/studentprofile.json:1]: I was not able to validate the below AC, is it present?
_BCSC only Revise text: Remove "here" from sentence and add "." after "up-to-date."_

Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/studentprofile.json:1]: My understanding from the below AC and from the image in the ticket is that the banner header should be replaced by "Information".

_Common to Both Revise info box header: "How to update your information" to "Information;_

**Review by andrewsignori-aot — COMMENTED** [review]
Please take a look at the comments.

File: sources/packages/forms/src/form-definitions/studentprofile.json [hunk 1]
```diff
@@ -1402,7 +1402,7 @@
               "value": ""
             }
           ],
-          "content": "<strong class=\"ml-2\">How to update your information</strong>\r\n<br />\r\n<span>\r\n  Your information above is taken from your B.C. Services Card. If the\r\n  information is incorrect, please update your information with\r\n  <a\r\n    class=\"primary-color\"\r\n    href=\"https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/your-card/change-personal-information\"\r\n    target=\"_blank\"\r\n  >\r\n    B.C. Services Card</a\r\n  >\r\n  first.\r\n</span>",
+          "content": "<strong class=\"ml-2\">How to update your Information</strong>\n<br />\n<span>\n  Your information above is taken from your BC Services Card account. If the\n  information is incorrect, please update your information with\n  <a\n    class=\"primary-color\"\n    href=\"https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/your-card/change-personal-information\"\n    target=\"_blank\"\n  >\n    BC Services Card account.</a></span>",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/studentprofile.json:1405]: Please add `rel='noopener noreferrer'` to the hyperlink.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍

**Review by andrepestana-aot — APPROVED** [review]
Looks good.

**Pull request merged**